shared/metrics/metrics.go

```diff
@@ -154,9 +161,9 @@ func NewPromMetrics() Metrics {
 		}, []string{}),
 		hplLatency: prom.NewHistogramFrom(stdprom.HistogramOpts{
 			Namespace: "solver",
-			Name:      "latency_per_hyperlane_message_seconds",
-			Help:      "latency for hyperlane message relaying, paginated by status, source and destination chain id (in seconds)",
-			Buckets:   []float64{30, 60, 300, 600, 900, 1200, 1500, 1800, 2400, 3000, 3600},
+			Name:      "latency_per_hyperlane_message_minutes",
+			Help:      "latency for hyperlane message relaying, paginated by status, source and destination chain id (in minutes)",
+			Buckets:   []float64{1, 5, 15, 30, 60, 120, 180, 240, 300},
```

Comment:
i think this was good as seconds, any reason in particular to switch to minutes?